inc/package.class.php

```diff
@@ -602,4 +604,27 @@ public function hook_entity_purge(CommonDBTM $item) {
       $package = new static();
       $package->deleteByCriteria(['entities_id' => $item->getField('id')], 1);
    }
+
+   /**
+    * @param array $input
+    * @param \ApkParser\Parser $apk
+    * @param string $destination
+    * @param string $uploadedFile
+    * @return array
+    */
+   protected function setApkManifest(array $input, \ApkParser\Parser $apk, $destination, $uploadedFile)
```

Comment:
I would name this method soemthing like getApkMetadata() or getApkManifest() because it gets metadata,  Set seems not appropriate here.

Also, if there is no identified need to inherit and share the class, it is better to declare it private first. If we need ti make it protected it is less risky  than making it protected and later change it to private.

Reply:
ok